pudb/source_view.py

```diff
@@ -222,3 +278,51 @@ def shipout_line():
                 PythonLexer(stripnl=False), UrwidFormatter())
 
         return result
+
+from enum import Enum
+class ParseState(Enum):
```

Comment:
Enum is Python 3-only, but pudb needs to support Python 2 for the moment. I think this should still work just fine if you make it a regular class for how you are using it.